test/parser_test.exs

```diff
@@ -12,9 +12,8 @@ defmodule Membrane.WAV.ParserTest do
   @input_path Path.expand("fixtures/input.wav", __DIR__)
   @reference_path Path.expand("fixtures/reference.raw", __DIR__)
 
-  defp perform_test(elements, links) do
-    pipeline_options = %Pipeline.Options{elements: elements, links: links}
-    assert {:ok, pid} = Pipeline.start_link(pipeline_options)
+  defp perform_test(structure) do
+    assert {:ok, _supervisor_pid, pid} = Pipeline.start_link(structure: structure)
```

Comment:
This is `Testing` so the return should be `pipeline` not a tuple

Reply:
According to documentation https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html#start_link/1 the return value is the same for Pipeline.start_link() and Testing.Pipeline.start_link().